backend/api/HostedServices/AutoSchedulingHostedService.cs

```diff
@@ -238,19 +227,23 @@ await MissionSchedulingService.ScheduleMissionRunFromMissionDefinitionLastSucces
             return;
         }
 
-        private void ReportMessageToSignalR(
-            string message,
-            string missionDefinitionId,
-            string installationCode
+        public async Task RemovePreviousDayscheduledJobsFromDatabase(
```

Comment:
```suggestion
        public async Task RemovePreviousDayScheduledJobsFromDatabase(
```

Reply:
Alternative name: ResetAutoScheduledJobs or ResetAutoScheduledJobsObjects. Shorter and more accurate as it resets all AutoScheduledJobs without actually checking if its last day